app/[id]+api.ts

```diff
@@ -54,7 +64,7 @@ export async function PUT(request: Request, { id }: { id: string }) {
     );
   }
 
-  const guest = getGuest(Number(id));
+  const guest = await getGuest(Number(id));
```

Comment:
because cookies are sometimes confusing in Expo / React Native, in future we should do 1 example with a database query that uses a session token

(let's not do this now though - this will require setting up auth in this project, which will be a bigger change than this PR / this round of changes should hold)